cmd/libs/go2idl/types/types.go

```diff
@@ -127,8 +160,20 @@ type Universe map[string]*Package
 // types will always be found, even if they haven't been explicitly added to
 // the map. If a non-existing type is requested, u will create (a marker for)
 // it.
-func (u Universe) Get(n Name) *Type {
-	return u.Package(n.Package).Get(n.Name)
+func (u Universe) Type(n Name) *Type {
+	return u.Package(n.Package).Type(n.Name)
+}
+
+// Function returns the canonical function for the given fully-qualified name.
+// If a non-existing function is requested, u will create (a marker for) it.
```

Comment:
Add (to function & variable & const, and corresponding methods in Package):

// If a marker is created, it's the caller's responsibility to finish construction of the function by setting Underlying to the correct type.